docs/basics/server.md

```diff
@@ -473,32 +473,37 @@ scriptEngines: {
-  - { className: org.apache.tinkerpop.gremlin.driver.ser.GraphSONMessageSerializerV3d0, config: { ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry] }}
+# JanusGraph sets default serializers. You need to uncomment the following lines, if you require any custom serializers.
+#
+# serializers:
+#   - { className: org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV3d0, config: { ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry] }}
+#   - { className: org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV3d0, config: { serializeResultToString: true }}
+#   - { className: org.apache.tinkerpop.gremlin.driver.ser.GraphSONMessageSerializerV3d0, config: { ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry] }}
 metrics: {
   slf4jReporter: {enabled: true, interval: 180000}}
 ```
 
 **There are several important parts to this configuration file as they relate to JanusGraph.**
 
-1.  In the `graphs` map, there is a key called `graph` and its value is
-    `conf/janusgraph.properties`. This tells Gremlin Server to
-    instantiate a `Graph` instance called "graph" and use the
-    `conf/janusgraph.properties` file to configure it. The "graph" key
-    becomes the unique name for the `Graph` instance in Gremlin Server
-    and it can be referenced as such in the scripts submitted to it.
-
-2.  In the `plugins` list, there is a reference to
-    `JanusGraphGremlinPlugin`, which tells Gremlin Server to initialize
-    the "JanusGraph Plugin". The "JanusGraph Plugin" will auto-import
-    JanusGraph specific classes for usage in scripts.
-
-3.  Note the `scripts` key and the reference to
-    `scripts/janusgraph.groovy`. This Groovy file is an initialization
-    script for Gremlin Server and that particular ScriptEngine. Create
-    `scripts/janusgraph.groovy` with the following contents:
+1. In the `graphs` map, there is a key called `graph` and its value is
+   `conf/janusgraph.properties`. This tells Gremlin Server to
+   instantiate a `Graph` instance called "graph" and use the
+   `conf/janusgraph.properties` file to configure it. The "graph" key
+   becomes the unique name for the `Graph` instance in Gremlin Server
+   and it can be referenced as such in the scripts submitted to it.
+
+2. In the `plugins` list, there is a reference to
+   `JanusGraphGremlinPlugin`, which tells Gremlin Server to initialize
+   the "JanusGraph Plugin". The "JanusGraph Plugin" will auto-import
+   JanusGraph specific classes for usage in scripts.
+
+3. Note the `scripts` key and the reference to
+   `scripts/janusgraph.groovy`. This Groovy file is an initialization
+   script for Gremlin Server and that particular ScriptEngine. Create
+   `scripts/janusgraph.groovy` with the following contents:
+
+4. Note the `serializers` list will be configured by an default set of serializers, 
```

Comment:
How can that happen if the user uses the default `bin/gremlin-server.sh` script from the Gremlin Server distributed by Apache TinkerPop as described here?
I think this section needs to mention our custom startup class and that it can be used by adapting the script which will then result in the automatic configuration of serializers. (In that case, the example output from starting the server also needs to be changed to include the JanusGraph ascii art.)